mock/ecs_client.go

```diff
@@ -91,29 +89,29 @@ type ECSContainerDefinition struct {
 	Name     *string
 	Image    *string
 	Command  []string
-	MemoryMB *int64
-	CPU      *int64
+	MemoryMB *int32
+	CPU      int32
```

Comment:
Why is memory an int pointer and CPU just an int?

Reply:
This made them match the types in [the ECS ContainerDefinition](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ecs@v1.27.1/types#ContainerDefinition). Definitely _could_ both be pointers, though.

Reply:
Huh, weird choice by the SDK then 🤷